charts/stable/nextcloud/questions.yaml

```diff
@@ -0,0 +1,322 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+# Include{containerBasic}
+# Include{containerAdvanced}
+
+  - variable: secretEnv
+    group: "App Configuration"
+    label: "Image Secrets"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: NEXTCLOUD_ADMIN_USER
+          label: "NEXTCLOUD_ADMIN_USER (First Install Only)"
+          description: "Sets the initial nextcloud's admin username, changing this variable after first launch will NOT change admin's username"
+          schema:
+            type: string
+            required: true
+            default: "REPLACETHIS"
+        - variable: NEXTCLOUD_ADMIN_PASSWORD
+          label: "NEXTCLOUD_ADMIN_PASSWORD (First Install Only)"
+          description: "Sets the initial nextcloud's admin password, changing this variable after first launch will NOT change admin's password"
+          schema:
+            type: string
+            private: true
+            required: true
+            default: "REPLACETHIS"
+  - variable: env
+    group: "App Configuration"
+    label: "Image Environment"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: TRUSTED_PROXIES
+          label: "Trusted Proxies (Advanced)"
+          description: "Sets nextcloud Trusted Proxies"
+          schema:
+            type: string
+            default: "172.16.0.0/16 127.0.0.1"
```

Comment:
we can fetch cluster CIDR using common now, so this just needs any additions to that.